<launch>

    <node pkg="me5413_perception" name="digit_recognition" type="digit_recognition.py" output="screen" />

</launch>